CMakeLists.txt

```diff
@@ -0,0 +1,164 @@
+cmake_minimum_required(VERSION 3.16.2)
+
+project(i3-ipc++ VERSION 0.8.0 LANGUAGES CXX)
+
+include(CMakePackageConfigHelpers)
+
+if (NOT DEFINED CMAKE_CXX_STANDARD)
+    set(CMAKE_CXX_STANDARD 17)
+    set(CMAKE_CXX_STANDARD_REQUIRED YES)
+endif()
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+#                              rapidjson                               #
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ #
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake/modules/")
+find_package(rapidjson REQUIRED)  # TODO
```

Comment:
You don't really need a hand written Find module for RapidJSON. It provides one already. Just use it instead: `find_package(RapidJSON REQUIRED)`